code/web/src/modules/subscription/api/actions.js

```diff
@@ -141,3 +142,14 @@ export function remove(variables) {
     }))
   }
 }
+
+//Update single date for availability
+export const updateAvailability = (date) => {
+  return dispatch => {
```

Comment:
Seeing dispatch everywhere had me curious so I went and did some research. It's interesting to see how an object is passed through a JS architecture.

Reply:
@ckccameron agreed! Dispatch is the path to get the action object from the action creator to the reducer in `state.js`.  I like to think of it as the messenger between user interaction and updates to the Redux store.